loris/resolver.py

```diff
@@ -80,11 +78,12 @@ def get_extra_info(self, ident, source_fp):
         Returns:
             dict: The dict of information to embed in info.json
         """
+        if not self.use_auth_rules:
+            return {}
         xjsfp = source_fp.rsplit('.', 1)[0] + "." + self.auth_rules_ext
         if exists(xjsfp):
-            fh = open(xjsfp)
-            xjs = json.load(fh)
-            fh.close()
+            with open(xjsfp) as fh:
+                xjs = json.load(fh)
```

Comment:
👍 

We could go one step further, and return straight out from inside the `with` block:

```python
with open(xjsfp) as fh:
    return json.load(fh)
```